hs_core/templates/search/search.html

```diff
@@ -1116,13 +1131,84 @@ <h4 class="panel-title">
                     $("#discover-page-options").html(pageOptionDiv);
 
                     initializeTable();
+                    $("#discover-load").hide();
+                },
+                failure: function (data) {
+                    console.error("Ajax call for updating list-view data failed");
```

Comment:
These calls are useful for debugging, but I'm not sure if they should be pushed to production. @pkdash, any thoughts?